docker/cami-medical-compliance/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM williamyeh/ansible:ubuntu16.04
+
+# or, for example, FROM ansible/ubuntu14.04-ansible:stable
+
+# Add playbooks to the Docker image. This requires that the build context is the
+# project root folder. Run build like:
+# `docker build -f docker/cami-store/Dockerfile .`
+COPY . /cami-project
```

Comment:
@iprunache I don't think this is a standard filesystem layout. Take a look here: https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard. It's quite an interesting read, IMO.